user/imodule.go

```diff
@@ -245,8 +247,12 @@ func (this *Module) Decode(em *ebpf.Map, b []byte) (result string, err error) {
 	}
 	return
 }
-
+func (this *Module) GetConf() IConfig {
+	return this.conf
+}
 // 写入数据，或者上传到远程数据库，写入到其他chan 等。
 func (this *Module) Write(result string) {
-	this.child.Write(result)
+	if result!=""{
+		this.child.Write(result)
```

Comment:
child module method `Write()` will replace parent method.  revert it ,and do in child module methon

Reply:
copy that